extension/src/index.ts

```diff
@@ -16,7 +22,7 @@ export async function activate(ctx: ExtensionContext) {
   }
 
   if (getConfig('fileNestingUpdater.autoUpdate')) {
```

Comment:
autoUpdateInterval is possibly undefined. 

I simply set it to fallback to its default value of 720 in that case.